### PR TITLE
MOHAWK: MYST: Reword accessibility option for Selenitic Age

### PR DESCRIPTION
**Pull request by lotharsm**
This is just a small patch to reword the new accessibility option in Myst's game options.

Opening as PR since english is not my first language and there might be better solutions.

### CODE REVIEW

**Comment by digitall** [comment]
@lotharsm : The original English for this option is a bit colloquial and fuzzy as in fuzzy logic is not really clear for non-specialist users. Your change is good in that respect, though you are losing the ~F~ which I think indicates the keyboard shortcut for control... Might want to add a similar one as this is about accessability.

**Comment by digitall** [comment]
@lotharsm : You can shorten English sentences much more than German so "Enable accessibility feature for the Selenitic Age" can be shortened to "Improve Selenitic Age puzzle accessibility". I would suggest rewording the description to "Allow solving Selenitic Age audio puzzles with more error margin".

**Comment by lotharsm** [comment]
@digitall Thank you very much for your suggestion which is way better than mine :-)

I adopted it and also brought back the hotkey. I went with "~a~" since it can be easily attached to "accessibility".

**Comment by digitall** [comment]
@lotharsm No problem! https://github.com/scummvm/scummvm/commit/e31558ae51ca999896beb5f0f4872971c0358cfd looks fine to me. +1 here to merge.

**Comment by bluegr** [comment]
Thanks for the suggestions guys :) This looks much better now. Let's merge it then
